### DISCUSSION TITLE
Why does jotai 2.x not supporting React 16 ?

### DISCUSSION THREAD

**Discussion by csr632** [pmndrs/jotai#1938] [Q&A] [answered]
I notice jotai 2.x bump its peerDependencies of React from `>=16.8` to `>=17.0.0`. Is it using react-17-only feature? Does it still support React 16?

**Comment by dai-shi** [comment, marked as answer]
We unofficially supports React 16 and run CI tests with the latest version of 16.
https://github.com/pmndrs/jotai/blob/10001a7b34bf2dbd0cdff28b0f14877a4707e509/.github/workflows/test-multiple-versions.yml#L31

We may drop the support in v2.x eventually, so it's not guaranteed.
